crates/runtime/src/runtime.rs

```diff
@@ -35,7 +35,7 @@ impl RuntimeBuilder {
         let mut engine_config = wasmtime::Config::default();
         engine_config.async_support(true);
         engine_config.epoch_interruption(true);
-        engine_config.memory_init_cow(false);
+        engine_config.memory_init_cow(true);
```

Comment:
Let's not set anything here and use the default instead?

Reply:
You're absolutely right, that's the better way to go here!